.cirrus.yml

```diff
@@ -638,6 +660,26 @@ upload_snap_task:
         - 'cd contrib/snapcraft && snapcraft && snapcraft push *.snap --release edge'
 
 
+test_docs_task:
+
+    only_if: >-
+        $CIRRUS_BRANCH != $DEST_BRANCH &&
+        $CIRRUS_CHANGE_MESSAGE =~ '.*CI:DOCS.*'
+    depends_on:
+        - "gating"
+        - "vendor"
+          #- "test_docs"
+          #- "varlink_api"
+          #- "build_each_commit"
+    stub_script: '/bin/true'
+
+
+#publish_docs_task:
```

Comment:
Why are these commented out?

Reply:
for future, if we ever want to add a task that would do something for publishing/preparing tasks, we can do this here.